client/src/domain/event/EventHero.tsx

```diff
@@ -55,47 +54,31 @@ const EventHero: React.FC<Props> = ({
       locale
     );
 
-    return [locationName, streetAddress, addressLocality].join(", ");
+    return [locationName, streetAddress, addressLocality]
```

Comment:
I have seen this getLocationStr pattern so many times that I think it should be an own component.